raml-for-jaxrs/example-proj/src/main/resources/api.raml

```diff
@@ -0,0 +1,39 @@
+#%RAML 1.0
+title: Ricks API
+version: v1
+baseUri: https://api.github.com
+mediaType: application/json
+
+types:
+  Rick: !include includes/rick.json
+
+uses:
+  RicksLib: includes/ricks-library.raml
+
+/ricks:
+  description: Crucial information about Ricks
+  get:
+    responses:
+        200:
+          body:
+            application/json:
+              type: RicksLib.Rick[]
+  post:
+    body:
+      application/json:
+        type: RicksLib.Rick
+    responses:
+        200:
+          body:
+            application/json:
+              type: RicksLib.Rick
+
+  /{slug}:
+    uriParameters:
+      slug: string
+    get:
+      responses:
+        200:
+          body:
+            application/json:
+              type: Rick
```

Comment:
why not re-sure `RicksLib.Rick` here?

Reply:
It's just a way to showcase that JSON Schema and RAML Data Type can be used alongside. `RicksLib.Rick` can be used in all places.

Reply:
Please let me know if I should remove JSON Schema and use `RicksLib.Rick` instead.

Reply:
would it generate similar java code?

Reply:
Looks like generated files (inside `raml-playground/raml-for-jaxrs/example-proj/target/generated-sources/raml-to-jaxrs-maven-plugin/example/jsonschema/`) are identical.

Reply:
In that case, let's use only the type `Lib.Rick`

Reply:
Ugh, I was wrong. On previous test everything work fine, but then I removed generated code folder completely and tried to install the project. And now looks like class `Rick` (inside `/raml-playground/raml-for-jaxrs/example-proj/target/generated-sources/raml-to-jaxrs-maven-plugin/example/jsonschema/Rick.java`) implementation is not generated when JSON Schema is not used :confused:

Reply:
Both classes had the same name (json and raml).  In the pull request I submitted to @postatum, I changed the name from Rick to JsonRick.  That said, I'm not convinced in a small example that we should be using JsonSchema: jsonschema is a bit of an outlier in RAML (arrays and such are not handled the same way...).  And the project itself has examples of this.

I've also made the project standalone:  no parent in the pom.xml, everything explicitly stated.